faidx.c

```diff
@@ -922,3 +922,33 @@ const char *fai_parse_region(const faidx_t *fai, const char *s,
 void fai_set_cache_size(faidx_t *fai, int cache_size) {
     bgzf_set_cache_size(fai->bgzf, cache_size);
 }
+
+char *fai_path(const char *fa) {
+    char *fai = NULL;
+    if (!fa) {
+        hts_log_error("No reference file specified");
+    } else {
+
+        char *fai_tmp = strstr(fa, HTS_IDX_DELIM);
+        if (fai_tmp) {
+            fai_tmp += strlen(HTS_IDX_DELIM);
+            fai = strdup(fai_tmp);
```

Comment:
Maybe `hts_log_error()` if this fails (all other failures get an error message of some sort),